scripts/cloudfront_routing.js

```
@@ -0,0 +1,24 @@
+ */
+function handler(event) {
+    const request = event.request;
+    const uri = request.uri;
+
+    if (uri === "/index.html") {
+        request.uri = "/";
+    } else if (uri === "/swagger.html") {
+        request.uri = "/swagger";
+    }
+    else if (uri.startsWith("/index.html?")) {
+        request.uri = "/" + uri.substring("/index.html".length);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)